samples/sample_common/src/sample_utils.cpp

```diff
@@ -152,7 +178,7 @@ void CSmplYUVReader::Reset()
     }
 }
 
-mfxStatus CSmplYUVReader::LoadNextFrame(mfxFrameSurface1* pSurface)
+mfxStatus CSmplYUVReader::LoadNextFrame(mfxFrameSurface1* pSurface, mfxU32 displayOrder/*= std::numeric_limits<mfxU32>::max()*/)
```

Comment:
why have you commented default? leftover from debug?

Reply:
removed